### PR TITLE
test(ticdc,dm): modify download-integration-test-binaries script

### LINKED ISSUE #11563: test(ticdc,dm): download_binaries failed when branch is not valid.
### What did you do?

run script `dm/tests/download-integration-test-binaries.sh branch` and `scripts/download-integration-test-binaries.sh`

### What did you expect to see?

download binaries successfully

### What did you see instead?

download binaries failed

### Versions of the cluster

master

### PR DESCRIPTION
**Pull request by wk989898**
### What problem does this PR solve?

Issue Number: close #11563 

### What is changed and how it works?
When branch is not valid, try to use `master` branch instead.

#### Tests 

 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

File: dm/tests/download-integration-test-binaries.sh [hunk 1]
```diff
@@ -41,16 +41,29 @@ function download() {
 	wget --no-verbose --retry-connrefused --waitretry=1 -t 3 -O "${file_path}" "${url}"
 }
 
+function get_sha1(){
+	local repo="$1"
+    local branch="$2"
+	file_server_url="http://fileserver.pingcap.net"
+	sha1=$(curl -s "${file_server_url}/download/refs/pingcap/${repo}/${branch}/sha1")
+	if [ $? -ne 0 ] || echo "$sha1" | grep -q "Error"; then
+		echo "Failed to get sha1 with repo ${repo} banch ${branch}: $sha1. use banch master to instead" >&2
```
Comment (lance6716) [inline dm/tests/download-integration-test-binaries.sh:50]: banch -> branch

**Review by lance6716 — APPROVED** [review]
linter should require you to format the shell scripts.

rest lgtm

**Comment by lance6716** [comment]
/hold

need to address comment

**Pull request merged**